package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fs-ext",
-  "version": "1.2.2",
+  "version": "1.3.0",
```

Comment:
I don't touch this file. I think it was synchronized with `package.json` during execution of `npm install` command.